motoman_driver/src/io_relay.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ * Copyright (c) 2020, Southwest Research Institute
```

Comment:
```suggestion
 * Copyright (c) 2016, Delft Robotics Institute
 * Copyright (c) 2020, Southwest Research Institute
```

seeing as this code is largely copied, would be nice to mention them (they pay my salary, so :))

Reply:
Hahaha fair point. I need to watch out for the copyrights (I get a little bit copy and paste happy)

Addressed in 001d34e and 4ec0248

Reply:
Thanks.

Do you not like the *Commit suggestion* feature?